apps/condo/domains/notification/constants/constants.js

```diff
@@ -29,6 +29,7 @@ const TICKET_STATUS_COMPLETED_TYPE = 'TICKET_STATUS_COMPLETED'
 const TICKET_STATUS_RETURNED_TYPE = 'TICKET_STATUS_RETURNED'
 const TICKET_STATUS_DECLINED_TYPE = 'TICKET_STATUS_DECLINED'
 const TICKET_COMMENT_ADDED_TYPE = 'TICKET_COMMENT_ADDED'
+const BILLING_RECEIPT_AVAILABLE_MANUAL_TYPE = 'BILLING_RECEIPT_AVAILABLE_MANUAL'
```

Comment:
You will use it in a future. Why you need manually suffix?

Reply:
There will be different text i future usages and there will be 3 types, not one